pages/admin/index.tsx

```diff
@@ -31,6 +31,7 @@ const Admin: NextPage = () => {
             ))}
           </select>
           <button className="text-blue-100">Export CSV</button>
+          <ApplicationsTable students={[]} />
```

Comment:
It's currently rendering like: 
<img width="991" alt="image" src="https://user-images.githubusercontent.com/44041989/201488804-6d5ea694-90fc-4786-9f87-006f843a8358.png">

`ApplicationTable` should be after the first `</div>` tag

Reply:
Please fix

Reply:
![image](https://user-images.githubusercontent.com/71536798/201527933-aee18f40-0d4e-4969-9e15-0dda78ef9bad.png)

Changed location of component.

Reply:
@lambo-liu